http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/LeoProfile.scala

```diff
@@ -95,6 +95,17 @@ private[leonardo] object LeoProfile extends MySQLProfile {
             res.fold(e => throw e, identity)
           }
         )
+    implicit val mapListColumnType: BaseColumnType[List[String]] =
+      MappedColumnType.base[List[String], String](
+        _.asJson.printWith(Printer.noSpaces),
```

Comment:
trying to understand why does this need to be in json format.
do you have an example of how this looks like when persisted to DB?

Reply:
I'm guessing it'll be stored in the DB like `["foo","bar","baz"]`

I think this probably makes sense for `APP.extraArgs`. But we shouldn't overuse this.. many 1-N DB relationships should use a join table.

Reply:
Yeah, it seemed limitting to model it another way
<img width="409" alt="Screen Shot 2021-03-05 at 10 07 06 AM" src="https://user-images.githubusercontent.com/6465084/110133739-a618a280-7d9a-11eb-8979-a9a9d85385d0.png">